pypeit/core/coadd.py

```diff
@@ -1235,8 +1264,9 @@ def scale_spec(wave, flux, ivar, sn, wave_ref, flux_ref, ivar_ref, mask=None, ma
 
     return flux_scale, ivar_scale, scale, method_used
 
-
-def compute_stack(wave_grid, waves, fluxes, ivars, masks, weights, min_weight=1e-8):
+# JFH This can easily accomodate lists. Replace the (nspec, nexp) arrays with a list of [(nspec1), (nspec2), ...] where
```

Comment:
Remove this comment now that lists are implemented?

Reply:
Removed.